lib/runner/variant-reassignment.js

```diff
@@ -25,14 +34,12 @@ export default class VariantReassignment {
       this._processTransaction(transaction)
   }
 
-  _processProductDraft (productDraft, existingProducts) {
-    const matchingProducts = this._selectExistingProductsUsingSkus(
-      productDraft, existingProducts
-    )
+  _processProductDraft (productDraft) {
+    const matchingProducts = this._isReassignmentNeeded(productDraft)
```

Comment:
This should not be needed right? Because here we already get productDrafts filtered by `_selectProductDraftsForReassignment`  method in the previous function.

Reply:
Yes, I removed it in the next branch, so I won't do it here.